enginetest/enginetests.go

```diff
@@ -2295,6 +2295,50 @@ func TestDropConstraints(t *testing.T, harness Harness) {
 	AssertErr(t, e, harness, "ALTER TABLE t1 DROP CONSTRAINT fk1", sql.ErrUnknownConstraint)
 }
 
+func TestWindowAgg(t *testing.T, harness Harness) {
```

Comment:
Looks like this won't work if you have a partition but not an order

Probably fine for a first draft, but add a skipped test

Reply:
ah i thought order was necessary, my bad

Reply:
for my reference
> `percent_rank` should be used with ORDER BY to sort partition rows into the desired order. Without ORDER BY, all rows are peers.

Reply:
@zachmu So it does actually work, somehow. The nil initialization casts to `rowNum = 1` by default in `EvalRow`. It also works when i just initialize `rowNum=1`, instead of nil. Do you have a preference with which I go with? nil seems like more memory efficient, but 1 makes default behavior more obvious.